service/src/main/java/bio/terra/cbas/dao/MethodVersionDao.java

```diff
@@ -134,8 +134,16 @@ public int updateOriginalWorkspaceId(UUID methodVersionId, UUID originalWorkspac
   }
 
   public int deleteMethodVersion(UUID methodVersionId) {
+    MapSqlParameterSource methodVersionIdParamSource =
+        new MapSqlParameterSource("methodVersionId", methodVersionId);
+
+    String deleteMethodVersionSql = "DELETE FROM %s WHERE method_version_id = :methodVersionId";
```

Comment:
Not a super strong opinion but this does feel a little funky to me and it took a while to work out what the intention was... 

I wonder how offended you'd be if I suggested just spelling out the two (almost) identical sql strings here - 
```java
String deleteGithubMethodVersionDetailsSql = "DELETE FROM github_method_version_details WHERE method_version_id = :methodVersionId";
String deleteMethodVersionSql = "DELETE FROM method_version WHERE method_version_id = :methodVersionId";
```